src/test_l1_tvSettings.c

```diff
@@ -12980,11 +12954,11 @@ void test_l1_tvSettings_positive_GetCurrentComponentSaturation (void)
 	result = TvInit();
 	UT_ASSERT_EQUAL_FATAL(result, tvERROR_NONE);
 
-	/* Step 02: Calling tvsettings GetCurrentComponentSaturation and expecting the API to return success */
+	/* Step 03: Calling tvsettings GetCurrentComponentSaturation and expecting the API to return success */
 	result = GetCurrentComponentSaturation((tvDataComponentColor_t)Configfile.componentColor.modeId[0],&saturation);
 	UT_ASSERT_AUTO_TERM_NUMERICAL(result, tvERROR_NONE);
 
-	/* Step 03: Calling tvsettings GetCurrentComponentSaturation and expecting the API to return success */
+	/* Step 04: Calling tvsettings GetCurrentComponentSaturation and expecting the API to return success */
```

Comment:
updating the steps number can be reverted

Reply:
Reverted